### ISSUE TITLE
Test 1016.1 has issues with data.table 1.16.0

### ISSUE THREAD

**Issue by aadler** [Rdatatable/data.table#6411]
This is probably related to #6403, but listing for completeness.

### Error message
```r
Running test id 1016.1          Test 1016.1 produced 1 warnings but expected 0
Expected: 
Observed: Some columns are type 'integer64' but package bit64 is not installed. Those columns will print as strange looking floating point data. There is no need to reload the data. Simply install.packages('bit64') to obtain the integer64 print method and print the data again.
```

### SessionInfo
```r
R Under development (unstable) (2024-06-20 r86794 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8  LC_CTYPE=English_United States.utf8    LC_MONETARY=English_United States.utf8 LC_NUMERIC=C                           LC_TIME=English_United States.utf8    

time zone: America/New_York
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] R.oo_1.26.0       R.methodsS3_1.8.2 data.table_1.16.0 R.utils_2.12.3   

loaded via a namespace (and not attached):
[1] compiler_4.5.0 tools_4.5.0   
```

**Comment by MichaelChirico** [comment]
thanks! yep this is closed by #6403. I guess we should add a note to the NEWS.

**Comment by MichaelChirico** [comment]
Closed by NEWS item in #6412
